apps/gdalalg_raster_color_map.h

```diff
@@ -0,0 +1,69 @@
+/******************************************************************************
+ *
+ * Project:  GDAL
+ * Purpose:  "color-map" step of "raster pipeline"
+ * Author:   Even Rouault <even dot rouault at spatialys.com>
+ *
+ ******************************************************************************
+ * Copyright (c) 2025, Even Rouault <even dot rouault at spatialys.com>
+ *
+ * SPDX-License-Identifier: MIT
+ ****************************************************************************/
+
+#ifndef GDALALG_RASTER_COLOR_MAP_INCLUDED
+#define GDALALG_RASTER_COLOR_MAP_INCLUDED
+
+#include "gdalalg_raster_pipeline.h"
+
+#include <limits>
+
+//! @cond Doxygen_Suppress
+
+/************************************************************************/
+/*                       GDALRasterColorMapAlgorithm                    */
+/************************************************************************/
+
+class GDALRasterColorMapAlgorithm /* non final */
+    : public GDALRasterPipelineStepAlgorithm
+{
+  public:
+    static constexpr const char *NAME = "color-map";
+    static constexpr const char *DESCRIPTION =
+        "Generates a RGBA or RGBA dataset from a single band, using a color "
```

Comment:
RGBA or RGBA dataset? What's the difference?

Reply:
fixed